packages/client/src/index.ts

```diff
@@ -19,9 +19,28 @@
 
 import { DefaultRoutingV1HttpApiClient } from './client.js'
 import type { AbortOptions } from '@libp2p/interface'
-import type { PeerInfo } from '@libp2p/interface/peer-info'
+import type { PeerId } from '@libp2p/interface/peer-id'
+import type { Multiaddr } from '@multiformats/multiaddr'
+import type { IPNSRecord } from 'ipns'
 import type { CID } from 'multiformats/cid'
 
+export interface PeerRecord {
+  Schema: string
```

Comment:
This is so we can use the `Schema` field as a type disambiguator:

```suggestion
  Schema: 'peer'
```

E.g. when `record` is a `Record` this lets us do:

```ts
if (record.Schema === 'peer') {
  // tsc now knows that `record` is a `PeerRecord`
}
```

Reply:
Oeh, Thanks. I searched for this and couldn't find it anywhere.